python/cudf/cudf/utils/dtypes.py

```diff
@@ -640,6 +640,24 @@ def _can_cast(from_dtype, to_dtype):
         return np.can_cast(from_dtype, to_dtype)
 
 
+def _to_default_dtype(dtype):
```

Comment:
Can we (1) rename this to `_maybe_convert_to_default_type` and (2) add a docstring for this function?

Reply:
Done.